ee/tabby-webserver/graphql/schema.graphql

```diff
@@ -108,6 +108,7 @@ type GithubRepositoryProvider {
   id: ID!
   displayName: String!
   applicationId: String!
+  connected: Boolean!
```

Comment:
@liangfung use this field to tell whether a connection has been successfully done.